AmahiAnywhere/AmahiAnywhere/Cells/DownloadsBaseCollectionCell.swift

```diff
@@ -111,5 +115,20 @@ class DownloadsBaseCollectionCell: SwipeCollectionViewCell{
         }
     }
     
+    func setUpPercentagelabel(offlineFile: OfflineFile, dateSizeLabel: UILabel) {
+        let size = offlineFile.getFileSize()
+        let date = offlineFile.downloadDate?.asString ?? ""
+        let progress = offlineFile.progress * 100
+        let dateSizeText = "\(Int(progress))% | \(size), \(date)"
```

Comment:
Thanks for your contribution. I would suggest a change to this line for the following reasons:

- The date does not matter much while downloading
- The line keeps on moving because the % is variable font and has variable digits
- The `|` keeps it a little busy

I would suggest
```
let dateSizeText = "\(size) · \(Int(progress))%"
```

Reply:
@cpg Thanks for your valuable feedback

1 I will not append the date while downloading.
2 I will add the percentage in last
3 i will add "." instead of "|"

Is this right?

Reply:
I made a precise suggestion of change for one line. If you like it, do it, if not, explain why and suggest otherwise.

Reply:
@cpg I liked your suggestion.
I am doing that way.